addons/account/wizard/account_secure_entries_wizard.py

```
@@ -0,0 +1,268 @@
+            wizard.chains_to_hash_with_gaps = [
+                {
+                    'first_move_id': chain['moves'][0].id,
+                    'last_move_id': chain['moves'][-1].id,
+                } for chain in chains_to_hash if 'gap' in chain['warnings']
+            ]
+
+            not_hashable_unlocked_moves = []
+            move_to_hash_ids = []
+            for chain in chains_to_hash:
+                not_hashable_unlocked_moves.extend(chain['not_hashable_unlocked_moves'].ids)
+                move_to_hash_ids.extend(chain['moves'].ids)
+            wizard.not_hashable_unlocked_move_ids = [Command.set(not_hashable_unlocked_moves)]
+            wizard.move_to_hash_ids = [Command.set(move_to_hash_ids)]
+
+    @api.depends('company_id', 'chains_to_hash_with_gaps', 'hash_date', 'not_hashable_unlocked_move_ids', 'max_hash_date', 'unreconciled_bank_statement_line_ids')
+    def _compute_warnings(self):
+        for wizard in self:
+            warnings = {}
+
+            if wizard.max_hash_date:
+                max_hash_date_string = format_date(self.env, wizard.max_hash_date)
+            else:
+                max_hash_date_string = _("today")
+            warnings['account_max_hash_date'] = {
+                'message': _("Posted entries are currently secured up to %s, inclusive.", max_hash_date_string),
+            }
+
+            if not wizard.hash_date:
+                wizard.warnings = warnings
+                continue
+
+            if wizard.unreconciled_bank_statement_line_ids:
+                ignored_sequence_prefixes = list(set(wizard.unreconciled_bank_statement_line_ids.move_id.mapped('sequence_prefix')))
+                warnings['account_unreconciled_bank_statement_line_ids'] = {
+                    'message': _("There are still unreconciled bank statement lines before the selected date. "
+                                 "The entries from journal prefixes containing them will not be secured: %(prefix_info)s",
```

We raise an error instead of ignoring, right?
```suggestion
                                 "The entries from journal prefixes containing them can not be secured: %(prefix_info)s",
```